R/get_cohorts_paths.R

```diff
@@ -1,34 +1,34 @@
-#' Demographics Cohort File Path
+#' Demographic cohorts lookup Path
 #'
-#' @description Get the full path to the Demographics Cohort lookup
+#' @description Get the path to the demographic cohorts lookup, there is one
+#' lookup per year.
 #'
-#' @param year The year for the cohorts extract
+#' @param year financial year in '1718' format
 #' @param ... additional arguments passed to [get_file_path()]
 #'
-#' @return The path to the Demographics Cohort lookup
-#' as a [fs::path()]
+#' @return The path to the demographic cohorts lookup as an [fs::path()]
 #' @export
-#' @family cohort lookup file paths
+#' @family cohorts file path
 #' @seealso [get_file_path()] for the generic function.
-get_demog_cohorts_path <- function(year, ...) {
-  demog_cohorts_path <- get_file_path(
+get_demographic_cohorts_path <- function(year, ...) {
+  demographic_cohorts_path <- get_file_path(
     directory = fs::path(get_slf_dir(), "Cohorts"),
-    file_name = glue::glue("Demographic_Cohorts_{year}.rds"),
+    file_name = glue::glue("demographic_cohorts_{year}.rds"),
```

Comment:
Due to this change to lowercase, there is one test now failing as the file 'does not exist' (due to capitalization in the folder)

Reply:
Argh, that's my fault as I thought we could change the naming of the R files without affecting any SPSS things. We could probably just drop the test which looks for the zsav file (we will at some anyway!)